_data/authors.yml

```diff
@@ -4,7 +4,7 @@ peter_krauss:
   name: Peter Krauß
   web: http://www.upday.com
   email: peter.krauss@upday.com
-  bio: "1337?"
+  bio: "Peter joined upday at the very beginning."
```

Comment:
I would say that "Peter joined upday at the very beginning." is strange to see just that at the end of the articles.
Henning has "Henning leads the technical team at upday as Director Engineering. "  and I "Florina passionately works at upday as a Senior Android Developer. ". What would you say about putting something around your title?